### ISSUE TITLE
Extensions should be updated so that they work

### ISSUE THREAD

**Issue by patrickocoffeyo** [aquifer/aquifer#164]
As of now, some extensions don't work correctly with the latest version of Aquifer and they do not support or use some of the new APIs that we have so graciously built, such as the run api and the env api. These extensions should be updated so that they take advantage of these latest greatest blazing new features, and so that they are compatible with the breaking API changes we have made in the 1.0.0 branch.

Basically, we need to make sure that these following extensions work:
- ~~`aquifer-git` - @generalredneck~~
- `aquifer-coder` - @JeffTomlinson 
- `aquifer-artifact` - @chasingmaxwell

**Comment by chasingmaxwell** [comment]
aquifer-git seems to be working without issue on its 1.0.0 branch, which is not the default branch so maybe that's why it seemed not to be working.

**Comment by chasingmaxwell** [comment]
Changed the default branch of aquifer-git to 1.0.0 to match the aquifer project.

**Comment by generalredneck** [comment]
I did run into a small snag... when we run `aquifer extension-add aquifer-git` it downloads the default version of `^0.3.0` which was the latest stable package. This may mean we need to add some documentation to tell users that they have to go in and change `.aquifer/package.json` to be the correct version of the extension that matches with this version. That's at least until 1.0.0 is released as stable for both aquifer and aquifer-git. Not sure if this should be an issue on aquifer-git or a general documentation thing we need to address in the wiki.

**Comment by JeffTomlinson** [comment]
`aquifer-coder` fixes and enhancements here: https://github.com/aquifer/aquifer-coder/pull/6

**Comment by chasingmaxwell** [comment]
@generalredneck 

> This may mean we need to add some documentation to tell users that they have to go in and change .aquifer/package.json to be the correct version of the extension that matches with this version.

This should be set in the "source" property for the extension in aquifer.json. Ideally the user should never have to touch anything inside the .aquifer directory.
